README.md

```diff
@@ -44,9 +48,15 @@ To use this sample form:
 
 1. Import the linked CRF into Rave for a subject of your choosing.
 2. Log in with the sample app using the credentials of the subject you chose.
-
 > You should see two forms, Form 1 and Form 2. Form 1 opens as one page. Form 2 opens as multiple pages.
 
+### Using self-registration and data capture functionality
+
+1. User with valid api token, you can register a user.
+2. After registration user can upload different type of files.
```

Comment:
These should be the steps to go through the process, just like the Rave section.

1. Use the in-app registration to enroll a user
2. Log in with the user that was just created
3. Select an image, which will be uploaded to AWS S3